### PR TITLE
Add all bumpversion options to the postrelease command.

### PR DESCRIPTION
**Pull request by mauritsvanrees**
This continues the work by @rnc from PR #393. That PR was waiting for a rebase or merge. And today I added the `--final` option to `bumpversion` so it made sense for me to include it for `postrelease` as well.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,6 +4,10 @@ Changelog for zest.releaser
 7.0.1 (unreleased)
 ------------------
 
+- Add the ``bumpversion`` options to the ``postrelease`` command.
+  This means ``feature``, ``breaking``, and ``--final``.
```
Comment (reinout) [inline CHANGES.rst:8]: Two without dashes, final *with* dashes?

**Review by reinout — APPROVED** [review]
One documentation comment.

File: CHANGES.rst (hunk 1 above)
Comment (mauritsvanrees) [reply CHANGES.rst]: fixed.

**Comment by mauritsvanrees** [comment]
I have released 7.1.0 with this.

**Comment by rnc** [comment]
@mauritsvanrees Thanks very much! ; apologies for missing the rebase request message.

**Pull request merged**